app/main.py

```diff
@@ -5,7 +5,7 @@ def get_human_age(cat_age: int, dog_age: int) -> list:
 
 
 def convert_to_human(
-        animal_age: int, first_year: int, second_year: int, each_year: int
+    animal_age: int, first_year: int, second_year: int, each_year: int
```

Comment:
There is a syntax error in the parameter list of the `convert_to_human` function. The parameters should be separated by commas, not a forward slash. Replace the `/` with a `,` to fix this issue.